projects/adi_vna_8ch/common/adi_vna_8ch_bd.tcl

```diff
@@ -0,0 +1,347 @@
+###############################################################################
+
```

Comment:
Please remove lines 2, 4, 6.
Also, for the copyright year, set it to 2023 only.